mplc/utils.py

```diff
@@ -122,7 +122,11 @@ def init_logger(debug=False):
     log_filter = MyFilter("INFO")
     logger.opt(depth=1)
     # Forward logging to standard output
-    logger.add(sys.stdout, enqueue=True, filter=log_filter, level=0)
+    logger.add(sys.stdout,
+               enqueue=True,
+               filter=log_filter,
+               format='<blue>{time:YYYY-MM-DD HH:mm:ss}</> | {level: <7} | {message}',
```

Comment:
While you are updating log format, any chance to reduce the blank (spaces) after the `level` part: 

```sh
# from
2021-01-15 11:16:20.755 | INFO     | mplc.multi_partner_learning:(...)
# to 
2021-01-15 11:16:20.755 | INFO | mplc.multi_partner_learning:(...)
```